releasenotes/notes/fix-django-str-tags-ad4f0b9e2d0eff12.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixed the handling of some Django tags that caused type errors.
```

Comment:
Should we be more specific here? I think it's worth being explicit about the template tag.